xgettext.js

```diff
@@ -164,7 +164,9 @@ XGettext.prototype._discoverMatches = function( parsedInput ) {
 
 	traverse( parsedInput.ast, {
 		CallExpression: function( node ) {
-			var functionName = ( node.callee.property ) ? node.callee.property.name : node.callee.name;
+			// Pull the resultingFunction out of (0, resultingFunction)()
+			var callee = ( node.callee.type === 'SequenceExpression' ) ? _.last( node.callee.expressions ) : node.callee;
```

Comment:
>However, in the SpiderMonkey spec you linked, the `ArrowExpression` is immediately before the `SequenceExpression`, and the formatting is quite poor and ambiguous.

Yes, this explains my confusion. Misread indeed.

After some thought, I think it's quite reasonable to support `SequenceExpression` as part of the core library here. However, do you think we ought to validate that the callee is a function before deriving its name? Other node types have a `name` property too. Here's an example failing test:

```js
it( 'should not match object properties that are the last element of a sequence (comma) expression', function() {
	var source = '(0, obj._)("Hello World!")',
		matches = new XGettext().getMatches( source );

	expect( matches ).to.deep.equal( [] );
});
```